topics/introduction/tutorials/galaxy-reproduce/tutorial.md

```diff
@@ -0,0 +1,312 @@
+
+
+# Create a history and load data into it
+
+Each analysis in Galaxy starts by creating a new analysis history and loading data into it. Galaxy supports a huge variety of data types and data sources. Different ways of bringing data into Galaxy are explained [here]({{site.baseurl}}/topics/galaxy-interface/tutorials/get-data/slides.html#1). To reproduce published results, the data needs to be loaded from the public repository where the authors have deposited the data. This is most often done by importing data via a web link. 
+
+
+> ### {% icon hands_on %} Hands-on: Create history
+>
+> 1. Make sure you start from an empty analysis history.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. **Rename your history** to be meaningful and easy to find. For instance, you can choose **Reproduction of published Galaxy results** as the name of your new history.
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+{: .hands_on}
+
+
+> ### {% icon comment %} Background about the dataset
+> The Iris flower data set, also known as Fisher’s or Anderson's Iris data set, is a multivariate dataset introduced by the British statistician and biologist Ronald Fisher in his 1936 paper ({% cite Fisher1936 %}).
+> Each row of the table represents an iris flower sample, describing its species and the dimensions in centimeters of its botanical parts, the sepals and petals.
+> You can find more detailed information about this dataset on its dedicated [Wikipedia page](https://en.wikipedia.org/wiki/Iris_flower_data_set).
+{: .comment}
+
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. {% tool [Import](upload1) %} the file `iris.csv` from [Zenodo](https://zenodo.org/record/1319069/files/iris.csv) or from the data library (ask your instructor)
+>
+>    ```
+>    https://zenodo.org/record/1319069/files/iris.csv
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 2. **Rename** {% icon galaxy-pencil %} the dataset to `iris`
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 3. Check the **datatype**
+>    - Click on the history item to expand it to get more information.
+>    - The datatype of the iris dataset should be `csv`.
+>    - **Change** {% icon galaxy-pencil %} the datatype *if* it is different than `csv`.
+>      - Option 1: Datatypes can be **autodetected**
+>      - Option 2: Datatypes can be **manually set**
+>
+>    {% snippet faqs/galaxy/datasets_detect_datatype.md datatype="datatypes" %}
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="csv" %}
+>
+> 4. Add an `#iris` tag {% icon galaxy-tags %} to the dataset
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+>    Make sure the tag starts with a hash symbol (`#`), which will make the tag stick not only to this dataset, but also to any results derived from it.
+>    This will help you make sense of your history.
+>
+{: .hands_on}
+
+> ### {% icon comment %} Different types of datasets
```

Comment:
I love the content of this box, it's super useful, but is it relevant here? The reviewer isn't uploading composite datatypes, are you envisioning it being "for someone who might be reproducing another, unrelated analysis?" Might it make sense to collapse it by default (e.g. make it a comment for people to expand if they need it?)

Reply:
yes yes and yes. How can I collapse the box?